kio-client-java-spring/src/main/java/org/zalando/stups/clients/kio/spring/DateTimeUtils.java

```diff
@@ -15,15 +15,18 @@
  */
 package org.zalando.stups.clients.kio.spring;
 
-import static java.time.format.DateTimeFormatter.ISO_OFFSET_DATE_TIME;
-
+import java.time.format.DateTimeFormatter;
 import java.time.temporal.TemporalAccessor;
 
 public final class DateTimeUtils {
 
-    private DateTimeUtils() { }
+    public static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSZZZ");
```

Comment:
please use the pattern with `XXX` as zone offset part. See JavaDocs at class `DateTimeFormatter`:

> Offset X and x: This formats the offset based on the number of pattern letters. One letter outputs just the hour, such as '+01', unless the minute is non-zero in which case the minute is also output, such as '+0130'. Two letters outputs the hour and minute, without a colon, such as '+0130'. Three letters outputs the hour and minute, with a colon, such as '+01:30'. Four letters outputs the hour and minute and optional second, without a colon, such as '+013015'. Five letters outputs the hour and minute and optional second, with a colon, such as '+01:30:15'. Six or more letters throws IllegalArgumentException. Pattern letter 'X' (upper case) will output 'Z' when the offset to be output would be zero, whereas pattern letter 'x' (lower case) will output '+00', '+0000', or '+00:00'